src/__tests__/helpers/reducer.change.js

```diff
@@ -187,7 +187,6 @@ const describeChange = (reducer, expect, { fromJS }) => () => {
       foo: {
         anyTouched: true,
         values: {
```

Comment:
in reducer is used function `deleteInWithCleanUp` to cleanup, from it tests I see that if an object has no children it also removes the parent. In this case you should also remove `values` object
```suggestion
```